didcomm/pack.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+from typing import List, Optional
+
+from didcomm.interfaces.did_resolver import DIDResolver
+from didcomm.interfaces.secrets_resolver import SecretsResolver
+from didcomm.types.algorithms import AnonCryptAlg, AuthCryptAlg, SignAlg
+from didcomm.types.plaintext import Plaintext
+from didcomm.types.types import JSON, DID, DID_OR_KID
+
+
+class Packer:
+    """Packer of plaintext DIDComm messages."""
+
+    def __init__(self,
+                 secrets_resolver: SecretsResolver,
```

Comment:
The arguments there are optional. There should be a way to register default secrets_resolver and did_resolver for all Packers.